pkg/component/controller/apiserver.go

```diff
@@ -52,9 +52,9 @@ type APIServer struct {
 	Storage                   manager.Component
 	EnableKonnectivity        bool
 	DisableEndpointReconciler bool
-	gid                       int
```

Comment:
So we never set the gid? This means that we have always run the processes as gid 'root'?

I think we should set the gid instead?

Reply:
K0s was never doing this. It only deals with users so far. What group would you pick here?